vhci_get_frame_number is called really often for USB cameras and this message
alone is responsible for major performance drop. Since the driver seems to
function ok, simply disable the message for now.

diff --git a/drivers/usb/usbip/vhci_hcd.c b/drivers/usb/usbip/vhci_hcd.c
index 7fbe19d..313a79b 100644
--- a/drivers/usb/usbip/vhci_hcd.c
+++ b/drivers/usb/usbip/vhci_hcd.c
@@ -928,7 +928,7 @@ static void vhci_stop(struct usb_hcd *hcd)
 
 static int vhci_get_frame_number(struct usb_hcd *hcd)
 {
-	pr_err("Not yet implemented\n");
+	//pr_err("Not yet implemented\n");
 	return 0;
 }
 
